enforcer/src/main/java/org/wso2/choreo/connect/enforcer/server/AuthServer.java

```diff
@@ -83,18 +88,38 @@ public static void main(String[] args) {
             // Create a new server to listen on port 8081
             Server server = initServer();
 
+            EnforcerConfig enforcerConfig = ConfigHolder.getInstance().getConfig();
             // Enable global filters
-            if (ConfigHolder.getInstance().getConfig().getAnalyticsConfig().isEnabled()) {
-                logger.info("analytics filter is enabled.");
+            if (enforcerConfig.getAnalyticsConfig().isEnabled() ||
+                    enforcerConfig.getMetricsConfig().isMetricsEnabled()) {
                 AccessLoggingService accessLoggingService = new AccessLoggingService();
                 accessLoggingService.init();
+                if (enforcerConfig.getMetricsConfig().isMetricsEnabled()) {
+                    //Initialize metrics
+                    MetricsManager.initializeMetrics(enforcerConfig.getMetricsConfig());
+                } else {
```

Comment:
Shall we check for analytics enabled prior to printing this log.

Reply:
Done

Reply:
Btw There can be a occasion where both is enabled. Hence it is not handled in the code right?

Reply:
Yes, we need to handled the part when both metrics, analytics are enabled. Fixed the issue in 45c855220c12a3238c90c7dad1f9a96afc80e427